velox/common/caching/tests/SsdFileTrackerTest.cpp

```diff
@@ -48,4 +48,24 @@ TEST(SsdFileTrackerTest, tracker) {
       tracker.findEvictionCandidates(kNumRegions, kNumRegions, pins);
   std::vector<int32_t> expected{0, 1, 4, 5, 6, 7, 8, 9};
   EXPECT_EQ(candidates, expected);
+
+  // Test large region scores.
+  tracker.clear();
+  for (auto region = 0; region < kNumRegions; ++region) {
+    tracker.regionRead(region, INT32_MAX);
+  }
+  // After 20 iterations, all region scores are set to INT64_MAX.
+  for (int i = 0; i < 20; i++) {
+    for (auto region = 0; region < kNumRegions; ++region) {
+      tracker.regionFilled(region);
+    }
+  }
+  // Bump up region scores once more, closer to UINT64_MAX.
+  for (auto region = 0; region < kNumRegions; ++region) {
+    tracker.regionRead(region, region * 1'000'000'000);
```

Comment:
Can we explicit check the region score is capped at max uint64_t by SsdFileTracker::regionScores? Thanks!